rtrlib/rtr/rtr.h

```diff
@@ -39,6 +48,35 @@ enum rtr_rtvals {
     RTR_INVALID_PARAM = -2
 };
 
+enum interval_range { 
+    BELOW_INTERVAL_RANGE = -1,
+    INSIDE_INTERVAL_RANGE = 0,
+    ABOVE_INTERVAL_RANGE = 1
+};
+
+enum interval_type {
+    EXPIRATION = 0,
+    REFRESH = 1,
+    RETRY = 2
+};
+
+/**
+ * @brief These modes let the user configure how received intervals should be handled.
+ */
+enum interval_mode {
+    /** Ignore appliance of interval values at all. */
+    IGNORE_ANY = 0,
```

Comment:
Should I adjust the comments for the rtr_socket_state enum as well, while I'm at it? Or are these not relevant for Doxygen?